iOS/ios_product_price_map_sample.csv

```diff
@@ -0,0 +1,4 @@
+product_id,price,currency,introductory_price,date,duration,introductory_price_duration

```

Comment:
Added the optional `introductory_price_duration` column to keep this sample complete